### PR TITLE
Level2 lights fix #53 

### LINKED ISSUE #53: [Level 2] Have Lights off to start
When the player starts they the lights should be off

### PR DESCRIPTION
**Pull request by WaynellLovell**

### CODE REVIEW

File: game/Assets/_Lvl2 Scripts/Lvl2GameController.cs [hunk 1]
```diff
@@ -8,15 +8,18 @@ public class Lvl2GameController : MonoBehaviour {
...
     public AudioSource backgroundmusic;
+    public GameObject Player;
+    public Transform SpawnPoint;
 
     // Use this for initialization
     void Start () {
         _lights = GameObject.FindGameObjectsWithTag("Light");
         foreach (GameObject light in _lights)
         {
             var source = light.GetComponent<Light>();
```
Comment (aagavin) [inline game/Assets/_Lvl2 Scripts/Lvl2GameController.cs]: Why are you using `var` here instead of the Object name?

Comment (WaynellLovell) [inline game/Assets/_Lvl2 Scripts/Lvl2GameController.cs]: when it works it works
i have tried using Light or GameObject but none of the two work but with var it works

Comment (aagavin) [inline game/Assets/_Lvl2 Scripts/Lvl2GameController.cs]: Have you tried something like `Lightsource = (Light) light.GetComponent<Light>();`

Comment (WaynellLovell) [inline game/Assets/_Lvl2 Scripts/Lvl2GameController.cs]: fixed

**Pull request merged**